infra/analytics/app-config/outputs.tf

```diff
@@ -17,3 +17,11 @@ output "build_repository_config" {
 output "environment_configs" {
   value = local.environment_configs
 }
+
+# This variable is slightly misnamed. It should really be called "has_migrations".
+# It controls whether or not the `run-database-migrations.sh` script tries to run database
+# migrations. The entire analytics application is going to have its schema controlled
+# via ETL jobs, so we don't need to run migrations in the same way as the API.
+output "has_database" {
+  value = false
+}
```

Comment:
I don't intend on updating the variable name. It seems fine to just document the issue here.